resolutions/numbers.md

```diff
@@ -3,3 +3,5 @@
 [2020-01](approved/2020-01.md) - Formation of Charter Maintenance Working Group
 
 [2020-02](proposed/2020-02.md) - Formation of Outreach Working Group
+
+[2021-01](approved/2021-01.md) - Clarify Steering Committee's responsibility to oversee business and marketing matters
```

Comment:
Right now this is in the `proposed` directory. I'm it will get moved if the resolution is approved?

Reply:
Correct.